src/plugins/intel_gpu/tests/test_cases/binary_convolution_gpu_test.cpp

```diff
@@ -180,6 +180,11 @@ class binary_convolution_test : public ::testing::TestWithParam<TestParams> {
 
 TEST_P(binary_convolution_test, conv) {
     auto& engine = get_test_engine();
+
+    //FIXME: https://github.com/openvinotoolkit/openvino/pull/12486
```

Comment:
`// Onednn was not validated for binary convolution: https://github.com/openvinotoolkit/openvino/pull/12486`

Reply:
Done.